.cubic/wiki/03-section-features/07-p-chat-ui.md

```
@@ -88,6 +88,13 @@ The system defines three distinct scopes for contextual interaction:
 
 Sources: [apps/backend/src/routes/contextChat.ts:43](../../../apps/backend/src/routes/contextChat.ts#L43), [apps/web/types.ts:471](../../../apps/web/types.ts#L471)
 
+### Contextual Follow-up Panel
```

markdownlint-cli2 (0.23.2)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)